upup/models/cloudup/resources/addons/networking.kuberouter/k8s-1.6.yaml.template

```diff
@@ -0,0 +1,106 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
Does kube-router work with k8s 1.5? If so we need a manifest, if not we need to prevent it being used with 1.5 and document that.

I am also not noticing rbac rules in the manifest, sorry that I missed that on my initial review.